src/main/resources/interface-specification.yml

```diff
@@ -1061,11 +1066,17 @@ components:
           type: string
           minLength: 10
           maxLength: 250
+        isFreeOfCharge:
+          type: boolean
+          default: true
```

Comment:
do we need a default?
this is an administrative declaration, shouldn't this be totally in charge of the user?

Reply:
this question is more for @beetlecrunch

Reply:
Let's remove this default value.